tensorflow/docs_src/get_started/get_started_for_beginners.md

```diff
@@ -303,7 +303,7 @@ and test sets respectively:
 
 ```python
     # Call load_data() to parse the CSV file.
-    (train_feature, train_label), (test_feature, test_label) = load_data()
+    (train_x, train_y), (test_x, test_y) = load_data()
```

Comment:
There are other uses of train_feature (such as line 285). I think we should change it the other way around, changing the single use of train_x to train_feature.

Reply:
Taking a closer look, train_x might be a feature dict, and train_feature might be a pandas dataframe. So they might not be the same thing (but could definitely use better names if this is the case)